bin/initoverlayfs-install

```diff
@@ -37,6 +37,30 @@ exec_erofs() {
     popd
     rm -f "${INITRAMFS_DIR}/initoverlayfs-$kver.img"
     mkfs.erofs $erofs_compression "${INITRAMFS_DIR}/initoverlayfs-$kver.img" ${INITRAMFS_DUMP_DIR}
+
+	generate_dm_verity_hash "${INITRAMFS_DIR}/initoverlayfs-$kver.img" "${INITRAMFS_DIR}/verity_table.img" $INITOVERLAYFS_CONF
+	if journalctl -b -o short-monotonic | grep -qi "dm-verity setup for initoverlayfs complete"; then
```

Comment:
optional: can we use long options instead of -b (--boot) and -o (--output), it makes easy to review.